platform/src/abstractions/EducationBenchmarking.Platform.Domain/DataObjects/FinancialPlanDataObject.cs

```diff
@@ -18,4 +18,5 @@ public record FinancialPlanDataObject
     [JsonProperty("totalNumberOfTeachersFte")] public decimal? TotalNumberOfTeachersFte { get; set; }
     [JsonProperty("educationSupportStaffCosts")] public decimal? EducationSupportStaffCosts { get; set; }
     [JsonProperty("useFigures")] public bool? UseFigures { get; set; }
+    [JsonProperty("Timetable")] public int? Timetable { get; set; }
```

Comment:
Json property name should be lower camel case.

Reply:
corrected in latest commit